index.html

```diff
@@ -173,11 +173,11 @@
 
 		<div class='stats' id='statsBox'>
 			<div class='stats-title'>
-				Active Programs in
+				Programs in
 				<br />
 				<span class="select" id="stats.districtType"></span> <span class="select" id="stats.districtName"></span> <span class="select" id="stats.districtSuffix"></span>
 				<br />
-				In <span class="select" id="stats.year"></span>
+				By <span class="select" id="stats.year"></span>
```

Comment:
This cumulative approach is much more intuitive. I wonder if we should change the word 'By' to 'As of', it's a small change but when I first read 'by' it seemed unclear if the programs on the map reflected the current year. I am fine to roll with this as-is and see what the Texans say.